libcxx.mixin

```diff
@@ -0,0 +1,10 @@
+build:
+  libcxx:
+    cmake-args:
+      - "-DCMAKE_C_COMPILER=clang-6.0"
+      - "-DCMAKE_CXX_COMPILER=clang++-6.0"
+      - "-DCMAKE_CXX_FLAGS=-stdlib=libc++ -D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS"
```

Comment:
This line should either quote the argument:

> `- "-DCMAKE_CXX_FLAGS='-stdlib=libc++ -D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS'"`

or split the two arguments on separate line:

> `- "-DCMAKE_CXX_FLAGS=-stdlib=libc++"`
> `- "-D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS"`

(not sure which of the two is intended).

Reply:
Added quotes - it's a c++ definition, not a CMake one